riocli/deployment/update.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2023 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import re
+from concurrent.futures import ThreadPoolExecutor
+from queue import Queue
+from typing import List
+
+import click
+from click_help_colors import HelpColorsCommand
+from rapyuta_io import Client, DeploymentPhaseConstants
+from rapyuta_io.clients.deployment import Deployment
+from yaspin.api import Yaspin
+
+from riocli.config import new_client
+from riocli.constants import Symbols, Colors
+from riocli.utils import tabulate_data
+from riocli.utils.spinner import with_spinner
+
+
+@click.command(
+    'update',
+    cls=HelpColorsCommand,
+    help_headers_color=Colors.YELLOW,
+    help_options_color=Colors.GREEN,
+)
+@click.option('--force', '-f', '--silent', is_flag=True, default=False,
+              help='Skip confirmation')
+@click.option('--update-all', '-a', is_flag=True, default=False,
+              help='Updates all deployments')
+@click.argument('deployment-name-or-regex', type=str, default="")
+@click.argument('workers', type=int, default=10)
```

Comment:
Workers should not be an argument. It should be a flag. Follow the convention in Apply command.